desktop/libs/Project/decoentry.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import { AppRegistry } from 'react-native'
+
+// modify this variable to move around components and screens
+let Entry = require('./index')
+
+// point to index if this is a release build
+if (!__DEV__) {
+  Entry = require('./index')
+}
+
+// Check if we're dealing with ES6-style imports
+if (typeof Entry === 'object' && Entry !== null && Entry.default) {
```

Comment:
I guess in practice it shouldn't matter... but technically shouldn't we check for Entry.__esModule here?